test/ComputeEngineIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import conceptual.api.ComputeEngine;
+import conceptual.api.ComputeEngineTwo;
+import process.api.DataStorageTestOnly;
+
+public class ComputeEngineIntegrationTest {
+	ComputeEngine ce = new ComputeEngine();
+	ComputeEngineTwo ce2 = new ComputeEngineTwo();
+	DataStorageTestOnly ds = new DataStorageTestOnly();
+
+	@org.junit.Test
+	public void validateInputFromCe() {
+		double[] testArr = { 1, 10, 25 };
+		assertEquals(ce.getInputSource("0100100001101001"), testArr, "CE Fail: Input Incorrect");
```

Comment:
use assertArrayEquals since you are comparing 2 arrays

Reply:
assertArrayEquals throws because the ce.getInputSource is returns InputSource, a generic placeholder type, but I'll leave a comment that it should be assertArrayEquals.